utilities/hamlet/Dockerfile

```diff
@@ -41,7 +41,9 @@ RUN chmod -R u+rwx /build/scripts && \
 
 RUN echo "Cloning images on $(date +%s)"
 RUN /build/scripts/build_hamlet.sh && \
-    /build/scripts/clone.sh
+    /build/scripts/clone.sh && \
+    echo "Updating Shell Prompt" && \
+    /build/scripts/shellprompt.sh
```

Comment:
Might be better to have this in
https://github.com/hamlet-io/docker-hamlet/blob/master/images/stretch/Dockerfile 

The utilities one is more around building hamlet on an image. For example apling doesn't use bash so this script might not work on there.

Reply:
Ahh, I hadn't considered someone overwriting the ARG and specifying a different underlying image. Makes sense though.

I'll update this, then happy to merge?

Reply:
The hamlet user is created in the utility script. I could implement it for root user in base image as you suggest and then in hamlet image perform a check to see if using bash, then `cp /root/.bashrc /home/hamlet/.bashrc` ?

Reply:
Yep that works

Reply:
updated and rebased. Ready for re-review.